frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -26,8 +26,8 @@ export const DashboardLoadingAndErrorWrapper = styled(
     );
   },
 )`
-  flex: 1 0 auto;
-
+  min-height: 100%;
+  height: 1px;
```

Comment:
This is needed for min-height in `DashboardStyled` to actually work. Alternatively, these may be reworked to use flex?